tests/test_Announce.py

```diff
@@ -0,0 +1,536 @@
+import time
+
+# Libs
+import discord
+import discord.ext.test as dpytest
+import mock
+import pytest
+from discord.ext import commands
+
+# Own modules
+import KoalaBot
+from cogs import Announce
+from tests.utils import TestUtilsCog
+
+# Varibales
+announce_cog: Announce.Announce = None
+utils_cog: TestUtilsCog.TestUtilsCog = None
+
```

Comment:
```suggestion
KoalaBot.is_dpytest = True
```
Add this to allow these tests to run on their own